homeassistant/components/fibaro/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I am missing a test where we try to add the mock config entry and then try to initialize a flow which should abort because the serial number is already used

Reply:
Agree: Status changed and comment added to quality_scale.yaml